website/src/css/customTheme.css

```diff
@@ -87,7 +87,7 @@ header h2 {
 }
 @media only screen and (min-device-width: 375px) and (max-device-width: 736px) {
   .home-splash .title {
-    font-size: 3.52rem;
+    font-size: 3.23rem;
```

Comment:
Why change this?

Reply:
> Why change this?

commit - "Fixed the size of hero slogan text, keyword overflowed earlier"

When "Performant" came up the keyword broke to a newline, leaving a single "A" in one line. To keep the text together, I decreased the font size.

Reply:
Talking about this - 

![image](https://user-images.githubusercontent.com/57267960/109945341-238dd580-7cfd-11eb-9265-ee3d3c8aea2d.png)

Also the CTAs get cropped out of view.

Reply:
I've fixed the problem, but haven't pushed the code yet

Reply:
Okay, is your solution same? If not, then maybe just push it in your next PR, I guess.

Reply:
Not the same, check out this.
https://github.com/apache/apisix-website/pull/241/commits/c39ee9ed13485fa604dea5a253592c283c5e4e09

Reply:
Should I make these changes in this push, if you don't mind.

Reply:
Please don't shrink down the font size, my purpose is to make the title fit the screen, make it looks less "left-aligned".

Reply:
Okay, then can you push your changes in your next pull request.

Reply:
https://github.com/apache/apisix-website/pull/241/commits

Reply:
Got it. Will make those CSS changes 👍

Reply:
@qier222 you are fine with this, **or is it work in progress.** 
![image](https://user-images.githubusercontent.com/57267960/109951139-55099f80-7d03-11eb-8cdf-ab6f80e5e1e5.png)

Can't really check on a real iphone, as I use android.

Reply:
Looks good on my build. Try clear your browser cache?
![image](https://user-images.githubusercontent.com/68148142/109952586-f6035500-7d19-11eb-94df-f4edfc72ffb6.png)

Reply:
`.home-splash .title {
  font-size: 5rem;
  font-weight: 800;
  color: #101827;
}`

`
@media only screen and (min-device-width: 376px) and (max-device-width: 736px) {
  .home-splash .title {
    font-size: 3.52rem;
  }
}
`

`
@media only screen and (min-device-width: 320px) and (max-width: 375px) {
  .home-splash .title {
    font-size: 3rem;
  }
}
`

No, I mean technically 375px is in neither of the media queries, so first style is applied, am I right. But, if you feel it looks fine on your system and phone, I'm fine then.

Reply:
Is it? Maybe I am having an issue that its not showing the recent load, or something 😕. Just cleared the cache too, I am using the Brave browser. Please check the main website again maybe.